build.gradle

```diff
@@ -15,7 +15,7 @@
  */
 
 plugins {
-    id 'net.wooga.plugins' version '2.2.2'
+    id 'net.wooga.plugins' version '3.0.0-rc.2'
```

Comment:
Don't update this. This will create a group change! We wanted to do this as a seperate upgrade round

Reply:
Whoops.

Reply:
Wait, I remember now why I did so in the first place. It wouldn't build until I updated it:

```
An exception occurred applying plugin request [id: 'net.wooga.plugins', version: '2.2.2']
> Failed to apply plugin class 'wooga.gradle.version.VersionPlugin'.
   > Could not create plugin of type 'VersionPlugin'.
      > Could not generate a decorated class for type VersionPlugin.
         > org/eclipse/jgit/errors/RepositoryNotFoundException has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

Reply:
This is the grgit issue. I think the latest 2.x solves this. So you can upgrade to this.

Reply:
`2.2.4` that is

Reply:
Yes, that fixed it. Thank you.